IbisVTK/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
-SET( IBIS_VTK_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/vtkExtensions ${CMAKE_CURRENT_SOURCE_DIR}/vtkMNI ${CMAKE_CURRENT_SOURCE_DIR}/vtkQt  )
-SET( IBIS_VTK_INCLUDE_DIR ${IBIS_VTK_INCLUDE_DIR} PARENT_SCOPE  )
-
 #================================
 # Define include dir for
 # dependent projects
 #================================
 
 add_subdirectory( vtkExtensions )
+target_include_directories( vtkExtensions PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/vtkExtensions
```

Comment:
Why is this not in vtkExtensions/CMakeLists.txt ?

Reply:
That was to replace lines

SET( IBIS_VTK_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/vtkExtensions ${CMAKE_CURRENT_SOURCE_DIR}/vtkMNI ${CMAKE_CURRENT_SOURCE_DIR}/vtkQt  )
SET( IBIS_VTK_INCLUDE_DIR ${IBIS_VTK_INCLUDE_DIR} PARENT_SCOPE  )

in the original  IbisVTK/CMakeLists.